.github/CHANGELOG.md

```diff
@@ -140,11 +140,14 @@
 * Add support for Scipy v1.14.
   [(#776)](https://github.com/PennyLaneAI/pennylane-lightning/pull/776)
 
+* Add pickle support for the `DevPool` object. 
```

Comment:
@Jaybsoni Isn't this support only for `lightning.gpu`? 

```suggestion
* Add pickle support for the `DevPool` object in `lightning.gpu`. 
```

Reply:
Yes! you are correct! I thought that was the only device with the attribute, better to be verbose!